public/api/conf/1.0/schemas/state-pension.json

```diff
@@ -0,0 +1,180 @@
+            },
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per week",
+              "example": "146.76"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per month",
+              "example": "638.14.1"
+            },
+            "annualAmount": {
+              "type": "number",
+              "description": "The forecasted amount of State Pension per year",
+              "example": "7657.73"
+            }
+          }
+        },
+        "maximum": {
+          "type": "object",
+          "properties": {
+            "yearsToWork": {
+              "type": "number",
+              "description": "The number of future qualifying years needed to achieve the maximum amount",
+              "example": "3"
+            },
+            "gapsToFill": {
+              "type": "number",
+              "description": "The minimum number of National Insurance gaps that need to be filled to achieve the maximum amount",
+              "example": "2"
+            },
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The maximum amount of State Pension per week",
+              "example": "155.65"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The maximum amount of State Pension per month",
+              "example": "676.8"
+            },
+            "annualAmount": {
+              "type": "number",
+              "description": "The maximum amount of State Pension per year",
+              "example": "8121.59"
+            }
+          }
+        },
+        "cope": {
+          "type": "object",
+          "properties": {
+            "weeklyAmount": {
+              "type": "number",
+              "description": "The amount of Contracted Out Pension Equivalent per week",
+              "example": "0"
+            },
+            "monthlyAmount": {
+              "type": "number",
+              "description": "The amount of Contracted Out Pension Equivalent per month",
+              "example": "0"
+            },
+            "annualAmount": {
+              "type": "number",
+              "description": "The amount of Contracted Out Pension Equivalent per year",
+              "example": "0"
+            }
+          }
+        }
+      }
+    },
+    "pensionAge": {
+      "type": "number",
+      "description": "The customer\u2019s age when they can claim State Pension",
+      "example": "64"
+    },
+    "pensionDate": {
+      "type": "string",
+      "description": "The date the customer can claim State Pension",
+      "example": "2018-07-06"
+    },
+    "finalRelevantYear": {
+      "type": "string",
+      "description": "The last full Tax Year before the customer\u2019s State Pension Date, for example 2017/18 with a date of 06/07/2018",
```

Comment:
The last full tax year before the customer\u2019s State Pension date. For example, 2017/18 with a date of 6 July 2018